packages/graphql/src/schema-model/Neo4jGraphQLSchemaModel.ts

```diff
@@ -86,7 +88,7 @@ export class Neo4jGraphQLSchemaModel {
     }
 
     public isCompositeEntity(entity?: Entity): entity is CompositeEntity {
-        return entity instanceof CompositeEntity;
+        return entity instanceof InterfaceEntity || entity instanceof UnionEntity;
```

Comment:
I thought the idea of having a composite entity is to avoid having both :thinking:

Reply:
The problem is they are very different and behave differently. All they have in common is a name and conceptually "dependant" concrete entities, however even these are very different: a union can only be made up of concrete entities, while an interface can also be implemented by interface entities (not unions).

Reply:
mmm. If that's the case, maybe we could scratch composite entities altogether :thinking: Or do you think it these are still useful?

Reply:
yes true.. As of now the CompositeEntity is just a kind of logical grouping, I was keeping it in case it proves useful as it doesn't seem to hurt but I could also remove it and add it back in case we want it at some point. Do you prefer to get rid of it?